socobo-inventory.html

```diff
@@ -0,0 +1,109 @@
+<link rel="import" href="elements.html">
+
+<dom-module id="socobo-inventory">
+  <style>
+    :host {
+      display: block;
+    }
+
+    .inventory {
+      margin: 8px;
+      padding: 8px;
+    }
+
+    .searchBox {
+      display: inline-block;
+      padding: 8px;
+      width: 100%;
+      @apply(--layout-horizontal);
+    }
+
+    .checkBoxAll {
+      margin-top: 32px;
+      margin-left: 16px;
+    }
+
+    .search-input {
+      @apply(--layout-flex);
+      margin-left: 32px;
+      margin-right: 32px;
+    }
+
+    .buttons {
+      float: right;
+    }
+
+    .itemCheckBox {
+      margin-right: 32px;
+      margin-top: 8px;
+    }
+
+    .overview {
+      padding: 0px;
+    }
+
+    .inventory-item-container {
+      margin: 8px 8px 0 0;
+      padding: 8px;
+      border-radius: 0;
+      border: 4px solid #ddd;
+    }
+
+    .item-name {
+      margin-top: 20px;
+    }
+  </style>
+
+  <template>
+    <paper-material vertical layout class="inventory" elevation="1">
+      <paper-material elevation="1">
```

Comment:
please use a toolbar; custom color is possible w/ --paper-toolbar-background and --paper-toolbar-color